src/App.svelte

```diff
@@ -1,45 +1,42 @@
 <script>
-	let visible = false;
-	import LeftComponent from './LeftComponent.svelte'
-	import RightComponent from './RightComponent.svelte'
-	import Footer from './Footer.svelte'
+    let visible = false;
+    import LeftComponent from './ScreenPage.svelte'
+    import RightComponent from './Download.svelte'
+    import Footer from './Footer.svelte'
 </script>
 
 
 <main>
-	<section class="bodyContainer">
-		<LeftComponent />
-		<RightComponent/>
-	</section>
-	<Footer />
+    <section class="bodyContainer">
+        <LeftComponent />
+        <RightComponent/>
+    </section>
```

Comment:
Nicely done.

Just one more thing. The name should coincide what you've changed it to. (`LeftComponent` and `RightComponent` should be changed to their corresponding component name)

Also, ScreenPage ain't a descriptive name. If you or someone else looks at the code in some weeks, You'll probably be wondering what `ScreenPage` or `Download` is